tablec.h

```diff
@@ -0,0 +1,153 @@
+#ifndef TABLEC_H
+#define TABLEC_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef TABLEC_SYMBOL_BUCKET
+#error "TABLEC_SYMBOL_BUCKET must be defined"
```

Comment:
Can't you just set some default values otherwise?

Reply:
That TableC branch is a fork of another branch, but allowing the modification of how it works. Because of being pratically the same, if the user didn't want to add the structs, he would end up using the original branch